src/styles.css

```
@@ -1,153 +1,266 @@
+}
 
-  text-align: center;
-  z-index: 9999;
-  box-shadow: 0px 0px 6px 0px rgba(0, 0, 0, 0.15);
+/* Toast essentials ------------------------------------------------------ */
 
-  cursor: pointer;
+:where([data-rct-toast]) {
+  --_rct-in: rct-enter-bottom;
+  --_rct-out: rct-exit-bottom;
+  position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)